source/errors/404.html.md.erb

```diff
@@ -0,0 +1,14 @@
+---
+title: 404 Page not found
+hide_from_sitemap: true
+hide_in_navigation: true
+index: false
```

Comment:
Would it make sense to use `layout: core` here so that the error pages don't get the navigation?

Reply:
Good shout 👍